api/data_explorer/controllers/export_url_controller.py

```diff
@@ -77,31 +78,31 @@ def _check_preconditions():
         raise BadRequest(error_msg)
 
 
-def _get_range_clause(column, value):
+def _get_range_clause(column, value, bucket_interval):
     arr = value.split('-')
     if len(arr) > 1:
         low = arr[0]
         high = arr[1]
```

Comment:
I'm not exactly sure what we're going for here, but based on the code below it looks like we're not using this 'high' variable. Lets delete it if so.